scripts/exregional_make_lbcs.sh

```diff
@@ -99,15 +99,6 @@ fi
 DATA="${DATA}/tmp_LBCS"
 mkdir_vrfy -p "$DATA"
 cd_vrfy $DATA
-
-if [ "${FCST_LEN_HRS}" = "-1" ]; then
-  CYCLE_IDX=$(( ${cyc} / ${INCR_CYCL_FREQ} ))
-  FCST_LEN_HRS=${FCST_LEN_CYCL[$CYCLE_IDX]}
-fi
-LBC_SPEC_FCST_HRS=()
-for i_lbc in $(seq ${LBC_SPEC_INTVL_HRS} ${LBC_SPEC_INTVL_HRS} $(( FCST_LEN_HRS+LBC_SPEC_INTVL_HRS )) ); do
-  LBC_SPEC_FCST_HRS+=("$i_lbc")
-done
```

Comment:
I have removed LBC_SPEC_FCST_HRS because RRFS figures out the forecast hour in a different way. Look lines 523-525

Reply:
This should not be removed. I think that AQM and SRW both use this option to find the LBCS. 

`EXTRN_MDL_FHRS` is sourced from the summary file written out by the get_extrnl_lbcs task, and I have to assume that @chan-hoo had a reason for not using that value, choosing this one instead in PR #613.

Reply:
Please leave this chunk of code for now and adopt the way that SRW does it. I believe this logic replaced the old logic that was in RRFS_dev1, and should not be reverted. Once that is done, I'm fine with this being merged.

Reply:
@christinaholtNOAA Note that if I leave that code as is, it will not work for RRFS (only SRW and AQM) because we are fetching more LBCS than the forecast length hour. I would have to either modify it to mirror that in `get_lbcs` which is:
```
  last_time=$((TIME_OFFSET_HRS + FCST_LEN_HRS))
  end_hr=$FCST_LEN_HRS
  if [ $BOUNDARY_LEN_HRS -gt $end_hr ]; then
     end_hr=$BOUNDARY_LEN_HRS
  fi
  # Download 0th hour lbcs if requested for it, mostly for DA
  if [ ${NEED_ALL_LBCS} = "TRUE" ]; then
    first_time=$((TIME_OFFSET_HRS))
  else
    first_time=$((TIME_OFFSET_HRS + LBC_SPEC_INTVL_HRS ))
  fi
  last_time=$((TIME_OFFSET_HRS + end_hr))


  fcst_hrs="${first_time} ${last_time} ${LBC_SPEC_INTVL_HRS}"
  ```
  Or use the hours  witten in the summary file, which should correspond to the above hours. I think this is a better strategy.  I have just run the make_lbcs step of AQM but did not re-build from scratch. It would be good if @chan-hoo can comment on this though.

Reply:
Maybe you can reach out to ChanHoo directly, instead of through comments here?

Reply:
@danielabdi-noaa @christinaholtNOAA, I apologize for my late response again. I was able to complete the NRT sample run of AQM successfully. I confirm that this change works for AQM correctly too.

Reply:
@chan-hoo Thanks for testing. Glad that the change works.